registry/manifest_test.go

```diff
@@ -0,0 +1,120 @@
+package registry
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/opencontainers/go-digest"
+)
+
+var (
+	fakeDigest = digest.FromString("sha256:0000000000000000000000000000000000000000000000000000000000000000")
+)
+
+func NewHandlerFunc(mediaType string) func(w http.ResponseWriter, r *http.Request) {
```

Comment:
I'm thinking we can make this lowercase since it's not meant to be used outside this package. Also, let's make it return a `http.Handler` so the caller doesn't have to do the conversion.

Reply:
https://github.com/stackrox/docker-registry-client/pull/20/commits/fd1ec934915c2a83f3103495ee227563ff7ef8c0